src/control/server/mgmt_system.go

```diff
@@ -272,6 +304,27 @@ func (svc *mgmtSvc) setFabricProviders(val string) error {
 	return system.SetMgmtProperty(svc.sysdb, fabricProviderProp, val)
 }
 
+func (svc *mgmtSvc) isGroupUpdatePaused() bool {
```

Comment:
Do we need to introduce some atomicity or locking to prevent any races with pause/resume and related workflows?

Reply:
The raft DB locks around getting/setting system attributes, so we should be good in that regard.